src/index.ts

```diff
@@ -10,16 +9,65 @@ export class RequestError extends Error {
   code?: number;
 }
 
-interface Headers {
-  [index: string]: string;
+export interface Headers {
+  // tslint:disable-next-line no-any
+  [index: string]: any;
```

Comment:
Why did you need to widen the type from `string` to `any`?

Note that this is semver major, but your PR is not marked as such.

Reply:
https://nodejs.org/api/http.html#http_request_setheader_name_value

The header value is marked `any` in the Node docs.
Node typings defines `http.OutgoingHttpHeaders` as `number|string|string[]|undefined`;

Reply:
I see, you're mirroring `request` library's `Headers` type, which is in fact `[index: string]: any`

Reply:
`Headers` was used only internally (not exported), and I believe these new exported types are a 1:1 to the types in `r.Options`. Do we really need a semver major here?